alacritty_terminal/src/config/mod.rs

```diff
@@ -115,6 +115,10 @@ pub struct Config<T> {
     #[serde(skip)]
     pub hold: bool,
 
+    // TODO: DEPRECATED
+    #[serde(default, deserialize_with = "failure_default")]
+    pub visual_bell: Option<BellConfig>,
```

Comment:
```suggestion
    visual_bell: Option<BellConfig>,
```

Reply:
It's required to throw deprecation warning.